<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1586</width>
    <height>976</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="2">
         <widget class="QPushButton" name="fwd">
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="4">
         <widget class="QLineEdit" name="img_folder">
          <property name="text">
           <string>/tmp/faces</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QPushButton" name="btn_img_folder">
          <property name="text">
           <string>Set Folder</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2" colspan="3">
         <widget class="QLabel" name="img">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Image</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" rowspan="2" colspan="2">
         <widget class="QLabel" name="gps_img">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QLabel" name="stats_img">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" rowspan="2">
         <widget class="QLabel" name="img_label">
          <property name="text">
           <string>Label</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="5">
         <widget class="QSlider" name="slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="2" colspan="2">
         <widget class="QLabel" name="img_info">
          <property name="text">
           <string>Image Info</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="bwd">
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="img_number">
          <property name="text">
           <string>0/0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Sequences</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_sequences">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>561</width>
          <height>771</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1586</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bwd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>bwd_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>873</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fwd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fwd_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>873</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_img_folder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>img_folder_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1306</x>
     <y>932</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>slider_moved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>884</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_sequences</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>listWidget_doubleclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fwd_clicked()</slot>
  <slot>bwd_clicked()</slot>
  <slot>img_folder_clicked()</slot>
  <slot>add_label_clicked()</slot>
  <slot>label_selected()</slot>
  <slot>jump_to_unlabeled()</slot>
  <slot>selected_labels_clicked()</slot>
  <slot>export_button_clicked()</slot>
  <slot>export_slider_value_changed()</slot>
  <slot>slider_moved()</slot>
  <slot>listWidget_doubleclick()</slot>
 </slots>
</ui>
